config/dependency_resolvers_conf.xml.sample

```diff
@@ -1,7 +1,10 @@
 <dependency_resolvers>
 <!-- the default configuration, first look for dependencies installed from the toolshed -->
   <tool_shed_packages />
-<!-- then look for env.sh files profile according to the "galaxy packages" schema -->
+<!-- then look for env.sh files in directories according to the "galaxy packages" schema.
+     These resolvers can take a base_path attribute to specify where to look for
```

Comment:
Its used in lib/galaxy/tools/deps/resolvers/resolver_mixins.py in the UsesToolDependencyDirMixin which is used by the galaxy_packages resolver.